pkg/component/kubernetes/apiserver/apiserver.go

```
@@ -597,6 +614,13 @@ func (k *kubeAPIServer) SetPodNetworkCIDRs(pods []net.IPNet) {
 	k.values.VPN.PodNetworkCIDRs = pods
 }
 
+func (k *kubeAPIServer) SetSeedPodNetwork(pods string) {
+	_, ipnet, err := net.ParseCIDR(pods)
+	if err == nil {
+		k.values.VPN.SeedPodNetwork = *ipnet
+	}
+}
```

Why do we not handle the potential error here? I guess we could either return the error to the caller or just do the parsing at the call-site instead and handle the error there. Ignoring the error does not seem like the best option.

---

I've changed it to `*net.IPNet` and added error handling in the functions that call it.